docs/developers/glutensupport.md

```diff
@@ -0,0 +1,62 @@
+---
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+# Gluten Support
+## Velox Backend
+
+[Gluten](https://github.com/oap-project/gluten) with velox backend supports Celeborn as remote shuffle service. Below introduction is used to enable this feature
+
+First refer to this URL(https://github.com/oap-project/gluten/blob/main/docs/get-started/Velox.md) to build Gluten with velox backend.
+
+When compiling the Gluten Java module, it's required to enable `rss` profile, as follows:
+
+```
+mvn clean package -Pbackends-velox -Pspark-3.3 -Prss -DskipTests
+```
+
+Then add the Gluten and Spark Celeborn Client packages to your Spark application's classpath(usually add them into `$SPARK_HOME/jars`).
+
+- Celeborn: celeborn-client-spark-3-shaded_2.12-0.3.0-incubating.jar
```

Comment:
There's a little doubt about that, we should use gluten compiled for `gluten-celeborn-XXX.jar` or `Celeborn-client-spark-3-shaded_2.12-XXX-incubating.jar`?

Reply:
The previous Gluten+Celeborn integrated ```celeborn-client-spark-3-shaded_2.12-XXX-incubating.jar``` into the Gluten compilation package, so users only need the Gluten package before.  After this pr (https://github.com/oap-project/gluten/pull/2530) was merged, the Celeborn package was removed from Gluten, so users need to put ```celeborn-client-spark-3-shaded_2.12-XXX-incubating.jar```  in the class path or add it into `$SPARK_HOME/jars`.